content/blog/2019-year-at-a-glance/index.md

```diff
@@ -49,7 +49,7 @@ It's impossible to highlight everything, but here are some of our favorite miles
+**Lots of providers!** We've added more than 20 new providers for managing resources across many cloud and service providers. This includes the existing core cloud providers for [AWS]({{< relref "/docs/intro/cloud-providers/aws" >}}), [Azure]({{< relref "/docs/intro/cloud-providers/azure" >}}), [GCP]({{< relref "/docs/intro/cloud-providers/gcp" >}}), and [Kubernetes]({{< relref "/docs/intro/cloud-providers/kubernetes" >}}), in addition to newly added providers [DigitalOcean]({{< relref "/docs/intro/cloud-providers/digitalocean" >}}), [Linode]({{< relref "/docs/intro/cloud-providers/linode" >}}), [OpenStack]({{< relref "/docs/intro/cloud-providers/openstack" >}}), [Packet]({{< relref "/docs/intro/cloud-providers/packet" >}}), and [vSphere]({{< relref "/docs/intro/cloud-providers/vsphere" >}}). Modern cloud infrastructures include components managed by SaaS or other service providers alongside and connected to your core cloud infrastructure, and for those, we have added support for [Aiven]({{< relref "/docs/intro/cloud-providers/aiven" >}}), [CloudAMQP]({{< relref "/docs/intro/cloud-providers/cloudamqp" >}}), [Cloudflare]({{< relref "/docs/intro/cloud-providers/cloudflare" >}}), [Consul]({{< relref "/docs/intro/cloud-providers/consul" >}}), [Datadog]({{< relref "/docs/intro/cloud-providers/datadog" >}}), [DNSimple]({{< relref "/docs/intro/cloud-providers/dnsimple" >}}), [Docker]({{< relref "/docs/intro/cloud-providers/docker" >}}), [Fastly]({{< relref "/docs/intro/cloud-providers/fastly" >}}), [F5 BigIP]({{< relref "docs/intro/cloud-providers/f5bigip" >}}), [GitLab]({{< relref "docs/intro/cloud-providers/gitlab" >}}), [Kafka]({{< relref "docs/intro/cloud-providers/kafka" >}}), [MySQL]({{< relref "docs/intro/cloud-providers/mysql" >}}), [New Relic]({{< relref "docs/intro/cloud-providers/newrelic" >}}), [Okta]({{< relref "docs/intro/cloud-providers/okta" >}}), [PostgreSQL]({{< relref "docs/intro/cloud-providers/postgresql" >}}), [RabbitMQ]({{< relref "docs/intro/cloud-providers/rabbitmq" >}}), [SignalFX]({{< relref "docs/intro/cloud-providers/signalfx" >}}), [Spotinst]({{< relref "docs/intro/cloud-providers/spotinst" >}}), and [Vault]({{< relref "docs/intro/cloud-providers/vault" >}}).
```

Comment:
I think all `relref` values must start with a `/`. cc @cnunciato 

For example, this in line `[Kafka]({{< relref "docs/intro/cloud-providers/kafka" >}})` should be `[Kafka]({{< relref "/docs/intro/cloud-providers/kafka" >}})`

Reply:
Yes, please start with `/` to avoid ambiguity.